apps/docs/content/components/Icon.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Icon
+source: https://github.com/primer/brand/tree/main/packages/react/src/Icon/Icon.tsx
+storybook: '/brand/storybook/?path=/story/components-icon--playground'
+description: Use the icon component to display Octicons with an optional background
+---
+
+import {PropTableValues} from '../../src/components'
+import {iconColors, namedIconSizes, numericIconSizes} from '@primer/react-brand'
+
+```js
+import {Icon} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Icon icon={ZapIcon} />
+```
+
+### Colors
+
+```jsx live
+<Stack direction="horizontal" gap={16} flexWrap="wrap" padding="none">
+  <Icon icon={RocketIcon} color="default" />
+  <Icon icon={RocketIcon} color="blue" />
+  <Icon icon={RocketIcon} color="coral" />
+  <Icon icon={RocketIcon} color="green" />
+  <Icon icon={RocketIcon} color="gray" />
+  <Icon icon={RocketIcon} color="indigo" />
+  <Icon icon={RocketIcon} color="lemon" />
+  <Icon icon={RocketIcon} color="lime" />
+  <Icon icon={RocketIcon} color="orange" />
+  <Icon icon={RocketIcon} color="pink" />
+  <Icon icon={RocketIcon} color="purple" />
+  <Icon icon={RocketIcon} color="red" />
+  <Icon icon={RocketIcon} color="teal" />
+  <Icon icon={RocketIcon} color="yellow" />
+</Stack>
+```
+
+### With background
+
+```jsx live
+<Stack direction="horizontal" gap={16} flexWrap="wrap" padding="none">
+  <Icon icon={RocketIcon} hasBackground color="default" />
+  <Icon icon={RocketIcon} hasBackground color="blue" />
+  <Icon icon={RocketIcon} hasBackground color="coral" />
+  <Icon icon={RocketIcon} hasBackground color="green" />
+  <Icon icon={RocketIcon} hasBackground color="gray" />
+  <Icon icon={RocketIcon} hasBackground color="indigo" />
+  <Icon icon={RocketIcon} hasBackground color="lemon" />
+  <Icon icon={RocketIcon} hasBackground color="lime" />
+  <Icon icon={RocketIcon} hasBackground color="orange" />
+  <Icon icon={RocketIcon} hasBackground color="pink" />
+  <Icon icon={RocketIcon} hasBackground color="purple" />
+  <Icon icon={RocketIcon} hasBackground color="red" />
+  <Icon icon={RocketIcon} hasBackground color="teal" />
+  <Icon icon={RocketIcon} hasBackground color="yellow" />
+</Stack>
+```
+
+### Sizes
+
+```jsx live
+<Stack direction="vertical">
+  {namedIconSizes.map((size) => (
+    <Icon
+      key={size}
+      icon={CopilotIcon}
+      size={size}
+      color="green"
+      hasBackground
+    />
+  ))}
```

Comment:
Can we tweak this example to be more obvious about size values please. `namedIconSizes` is an import in the mdx file and docs-only reader can't see what the allowed values are. We can keep the prop table values the same though.

![Screenshot 2024-12-02 at 13 46 59](https://github.com/user-attachments/assets/ad9c2462-45c4-4c4c-bc8a-bee4b6916533)


```suggestion
  <Icon icon={CopilotIcon} size="small" />
  <Icon icon={CopilotIcon} size="medium" />
  <Icon icon={CopilotIcon} size="large" />
```

Reply:
Ah, an oversight on my part there, thanks.